scripts/custom_lint_checks.py

```diff
@@ -311,6 +327,14 @@ def check_functiondef_returns(self, node, node_doc):
                 node=node)
 
     def check_functiondef_yields(self, node, node_doc):
+        """Checks whether all yields in a function definition are documented.
```

Comment:
A better way to phrase this would be:
"Checks whether a function documented with a yield value actually has a yield statement in its definition"

Reply:
done.